tests/sentry/runner/commands/test_configoptions.py

```diff
@@ -0,0 +1,41 @@
+from functools import cached_property
+
+import pytest
+from django.core.cache.backends.locmem import LocMemCache
+
+from sentry.options import OptionsManager
+from sentry.options.store import OptionsStore
+from sentry.runner.commands.configoptions import configoptions
+from sentry.testutils import CliTestCase
+
+
+class ConfigOptionsTest(CliTestCase):
+    command = configoptions
+
+    @cached_property
+    def store(self):
+        c = LocMemCache("test", {})
+        c.clear()
+        return OptionsStore(cache=c)
+
+    @cached_property
+    def manager(self):
+        return OptionsManager(store=self.store)
+
+    @pytest.fixture(autouse=True)
+    def setup(self):
+        self.store.flush_local_cache()
+
+    def test_get(self):
+        # symbolserver.options: {'url': 'http://127.0.0.1:3000'}
+        rv = self.invoke("get", "symbolserver.options")
```

Comment:
```
get("symbolserver.options")
```
?

Why do you need to call the method with the invoke command ?

Reply:
It is calling the command line argument. It invokes the click command